MCLauncher/MainWindow.xaml.cs

```diff
@@ -93,13 +113,13 @@ public partial class MainWindow : Window, ICommonVersionCommands {
             }
         }
 
-        public ICommand LaunchCommand => new RelayCommand((v) => InvokeLaunch((Version)v));
+        public ICommand LaunchCommand => new RelayCommand((v) => InvokeLaunch((WPFDataTypes.Version)v));
```

Comment:
Unrelated change.

Reply:
My IDE threw a fit without me doing this, I don't recall the specifics as this was two weeks ago but I believe not doing this caused some sort of namespace conflict.